Yosemite/YosemiteTests/Tools/Media/MediaTypeTests.swift

```diff
@@ -61,11 +61,17 @@ final class MediaTypeTests: XCTestCase {
     }
 
     func testInitWithVideoMimeType() {
-        let videoMimeTypes = [
+        var videoMimeTypes = [
             "video/mp4", "video/x-msvideo", "video/mpeg", "video/3gpp", "video/3gpp2",
             // 3gpp/3gpp2 audio is considered as video on iOS
             "audio/3gpp", "audio/3gpp2"
         ]
+        if #available(iOS 14.4, *) {
+            videoMimeTypes += [
+                "video/webm", // iOS 14.4 supports WebM
+                "audio/webm"  // audio/webm also results as a video type
+            ]
+        }
```

Comment:
These tests started failing because of `video/webm` and `audio/webm`. I used `#available(iOS 14.4, *)` to ensure a smooth transition for folks that are still using Xcode < 12.5

Reply:
@jaclync GitHub tells me you created this file? Does this change make sense to you?

Is this something that could impact the production code, too?

Reply:
Jaclyn is AFK, then I reviewed the code in her place.

It seems that we can exclude `video/webm` and `audio/webm` if possible for every system version, since also if we will be able to upload it, they will not be saved into a WooCommerce product. I did the test also on the web, and I can confirm that it seems a format not supported by WooCommerce/WordPress.

About the video with `webm` extension:
- It's uploaded correctly.
- It will be not added to the product gallery.
- Also from Wordpress, if you try to add it inside the description of the product, from Safari the video is not reproducible.

Reply:
I see... 🤔 I might open a new PR on top of this one to address this change, just to keep the conversation around it localized.

_Stay tuned_

Reply:
@mokagio just wanted to follow up on this – AFAIK we still need to refactor this test, right?